test/e2e/manifests/datadog-agent-minimum.yaml

```diff
@@ -27,3 +26,23 @@ spec:
           env:
             - name: DD_SKIP_SSL_VALIDATION
               value: "true"
+      env:
+        - name: DD_CLUSTER_NAME
+          valueFrom:
+            configMapKeyRef:
```

Comment:
why aren't we using `global.clusterName`? is it because cluster name is dynamic and we save it in CM and let kubernetes bind it when pod is created?

Reply:
yes, exactly. we should be able to remove the configMap when this test-infra-definitions [PR](https://github.com/DataDog/test-infra-definitions/pull/915) is merged. 
We'll be able to use the [NewDDAWithOperator](https://github.com/DataDog/test-infra-definitions/pull/915/files#diff-3351a1f7844763607831500fa37b0514622841a8d129ce78d98edf773329d9dcR16) method, which eventually sets `global.clusterName` [here](
https://github.com/DataDog/test-infra-definitions/pull/915/files#diff-78515e291708dc79576a9f35bcc7c9ede57ed78e21205260ae0194f769fdf317R106-R107).